src/basic/process-util.h

```diff
@@ -141,6 +141,11 @@ int must_be_root(void);
 
 pid_t clone_with_nested_stack(int (*fn)(void *), int flags, void *userdata);
 
+/* 💣 Note that FORK_NEW_USERNS + FORK_NEW_MOUNTNS should not be called in threaded programs, because they
+ * cause us to use raw_clone() which does not synchronize the glibc malloc() locks, and thus will cause
+ * deadlocks if the parent uses threads and the child intends to do memory allocation. Hence: if the parent
+ * is threaded these flags may not be used. Or conversely, if the forked of child uses malloc() these flags
+ * may not be used. 💣 */
```

Comment:
This is worded strangely. "Conversly" would imply an opposite case, but both cases are the same. So maybe just say "Those flags cannot be used if the parent uses threads or the child uses malloc."